lib/database.js

```diff
@@ -54,6 +54,7 @@ var CrudDriver = function(opts) {
     insert: insert,
     update: update,
     delete: deleteItem,
+    procedure: opts.procedure,
```

Comment:
Perhaps I wasn't really clear enough here, the procedure function shouldn't even be exposed as part of the Crud driver if it isn't available, what's the point?